src/systems/ackermann_steering/AckermannSteering.cc

```diff
@@ -505,12 +505,14 @@ void AckermannSteering::PreUpdate(const UpdateInfo &_info,
   GZ_PROFILE("AckermannSteering::PreUpdate");
 
   // \TODO(anyone) Support rewind
-  if (_info.dt < std::chrono::steady_clock::duration::zero())
-  {
-    gzwarn << "Detected jump back in time ["
-        << std::chrono::duration_cast<std::chrono::seconds>(_info.dt).count()
-        << "s]. System may not work properly." << std::endl;
-  }
+  
+if (_info.dt < std::chrono::steady_clock::duration::zero())
```

Comment:
can you update the indentation to match what it was before (2 spaces)?

Reply:
Fixed.